### ISSUE TITLE
Readme could use better orientation

### ISSUE THREAD

**Issue by aarongerber** [G-Wang/WaveRNN-Pytorch#11]
Perhaps I am the issue (as I am new to programming and wavernn), but your instructions are not clear on how to:

- Use the Pretrained Checkpoints
- Generate audio files from checkpoints
- Create your own data set to train from

A generating tool would be amazing (user could put large mp3/wav files in a folder with a .txt with sentences it would do all the work to generate sentences from the files... and/or give you warnings if the audio data wasn't sufficient (creating a data set by dividing large audio files into 5-10 seconds where there are moments of silence is not easy)

- I would love to have a link to a YouTube video that shows how you got Obama and Bernie setup. Such a fun example. Made me laugh.

I really appreciate you sharing your work; please do not take this issue as a lack of appreciation

**Comment by elnursamedzade** [comment]
Yes you are right. It is a very nice project but there is a lack of teaching.
